### ISSUE TITLE
Ability to view historical Data of unclaimed Plot

### ISSUE THREAD

**Issue by Never2old4minecraft** [MODNMETL/VirtualRealty#29]
Big thanks for the plugin, bought the plot panel and great addition. Is there a plugin feature to enable in game admin visibility of historical plot data of unclaimed plot. i.e who owned unclaimed plot before its became unclaimed or lease expired?
Can this be done or  considered as a feature to be included in future updates?

**Comment by MODNMETL** [comment]
Yes, you can simply use either the player-level '/plot info' command while standing in a plot to see who owns/owned it and when the lease expires/expired, or for more information on the plot you can use the admin-level '/vrplot info' command, either while standing in the plot concerned or by appending its plotID to the command and you can even see who (which admin) assigned the plot to the owner if it wasn't created/claimed using a plot claim item.

**Comment by Never2old4minecraft** [comment]
Hey, thanks so much for the rapid reply. I must be doing something wrong, this pic shows plot 10003, yesterday it was owned by a player, unclaimed overnight and when I use /vrplot info command whilst standing in the plot it shows no history of who the owner was or who assigned it. I assigned it yesterday to the player. any idea
<img width="1014" alt="Screenshot 2023-08-01 at 9 04 11 pm" src="https://github.com/MODNMETL/VirtualRealty/assets/141136171/8b9bbf3c-24a3-4522-8fb4-2e27a842cc10">

**Comment by Never2old4minecraft** [comment]
oh disregard my last, this plot was created with a claim item and that is why it likely doesn't show history

**Comment by Never2old4minecraft** [comment]
now this plot was created using a claim item so that's why I can't see me as the assigned by person, but I can't see the history of who owned it previously. Is this because of the claim item being used also

**Comment by MODNMETL** [comment]
No if a player claims an available plot with plot claim item (the plot claim item has to be of the same 'Size' type) the vrplot info command, or indeed the /plot info command will show you who claimed it whether they are still the active owner or not (e.g. if the lease has expired since)

Are you sure they claimed it successfully? Do you have discord at all? We could probably help more in there https://discord.gg/FkfBejaSEW
